dom0/sd-app-files.sls

```diff
@@ -18,3 +18,18 @@ install-securedrop-client-package:
       - securedrop-client
     - require:
       - sls: fpf-apt-test-repo
+
+install-securedrop-log-package:
+  pkg.installed:
+    - pkgs:
+      - securedrop-log
+    - require:
+      - sls: fpf-apt-test-repo
+
+sd-rsyslog-for-sd-app:
```

Comment:
Do we want this config to be there before or after installing the package? Do we need to restart the service afterwards?

Reply:
either will do, I am enabling the service afterwards.